content/docs/studio/user-guide/model-registry/download-models.md

```diff
@@ -0,0 +1,23 @@
+# Download models
+
+If your model file is DVC-tracked, you can download any of its registered
+versions using [dvc get](https://dvc.org/doc/command-reference/get).
+
+```cli
+$ dvc get ${GIT_REPO} ${MODEL_PATH} --rev ${GIT_REV}
+```
+
+For example, one of the models in the
+[bank customer churn demo project](https://github.com/iterative/demo-bank-customer-churn)
+is `lightgbm-model`. You can find its registered versions in the
+[list of Git tags](https://github.com/iterative/demo-bank-customer-churn/tags).
+
+Suppose you want to dowload the model file for the `lightgbm-model@v2.0.1`
+version. You can use the following dvc command:
+
+```cli
+$ dvc get https://github.com/iterative/demo-bank-customer-churn .mlem/model/lightgbm-model --rev lightgbm-model@v2.0.1
+```
```

Comment:
in case it doesn't fit the screen
```suggestion
```cli
$ dvc get https://github.com/iterative/demo-bank-customer-churn \
    .mlem/model/lightgbm-model \
    --rev lightgbm-model@v2.0.1
```
```